packages/sdk-codegen/src/codeGenerators.spec.ts

```diff
@@ -62,18 +63,18 @@ describe('generator factory', () => {
     it('returns generator by file extension', () => {
       let actual = findGenerator('.ts')
       expect(actual).toBeDefined()
-      expect(actual?.language).toEqual('Typescript')
+      expect(actual?.language).toEqual(typeScript)
```

Comment:
I'm in favor of your change, but maybe for consistency re-do the `"Csharp"` and `"Kotlin"` magic strings in the `describe` block too?

Reply:
Done